src/clauses/Merge.ts

```diff
@@ -56,12 +56,21 @@ export class Merge extends Clause {
         this.onCreateClause = new OnCreate(this);
     }
 
+    /**
+     * @deprecated Use {@link onCreateSet} instead
+     */
     public onCreate(...onCreateParams: OnCreateParam[]): this {
         this.onCreateClause.addParams(...onCreateParams);
```

Comment:
How do you feel about adding a console message when this is used saying to use `onCreateSet` instead?